ui/app/components/secure-variable-form.hbs

```diff
@@ -1,8 +1,8 @@
 {{did-update this.onViewChange @view}}
 {{did-insert this.establishKeyValues}}
 <form class="new-secure-variables" autocomplete="off" {{on "submit" this.save}}>
-  <div>
-    <label>
+  <div style="display: flex">
+    <label style="flex-grow: 5">
```

Comment:
@philrenaud I have some bigger priorities than doing guesswork on CSS, can we just ticket this and come back to it later? I noticed you're drifting from our normal CSS pattern in favor of a grid format for the row below this, but I don't have the bandwidth to learn CSS Grid.

I care more about addressing the `403` error that I reported in Slack along with the saving collision issue and testing/refactoring.

Reply:
Regardless of what we use here, assigning the styles inline is an approach that doesn't scale. Let's move these to the rest of the secure variables scss file